tests/conftest.py

```diff
@@ -29,3 +30,14 @@ def base_data_dir():
 @pytest.fixture
 def locations():
     yield FIXTURE_DIR / "sampleNodeLocations.json"
+
+
+@pytest.fixture(autouse=True, scope="session")
+def teardown_remove_data():
+    """Remove test output created during testing.
+
+    Datasets defined in data_pipeline_inputs/config.yaml can't be handled with
+    pytest's tmp_path, so are cleaned up here.
+    """
+    yield
+    shutil.rmtree(FIXTURE_DIR / "data_pipeline_inputs" / "output")
```

Comment:
I think the access log is stored in `FIXTURE_DIR / "data_pipeline_inputs"`: `access*.yaml`. If we could delete those too it'd be nice.

Reply:
> I think the access log is stored in `FIXTURE_DIR / "data_pipeline_inputs"`: `access*.yaml`. If we could delete those too it'd be nice.

`config.yaml` indicates an `access.log` file might be created, but I've yet to see one appear with local testing. I'm happy to add this speculatively, though.

Reply:
I think they are supposed to be called `access-*.yaml`. It's odd that they are not showing up there. Maybe they are getting created somewhere else?